index.html

```diff
@@ -112,91 +114,132 @@ <h1 style="margin-top: 10px">Smart figures with metadata</h1>
-              NB: For python 2.6, you'll need Reportlab 2.7: <code>$ pip install reportlab==2.7</code>
+              <code style="width:650px">
+                $ pip install reportlab
+              </code>
             </li>
-            <li>Optional: Figure legends can be formatted using Markdown syntax. To see this correctly in the
-              exported PDF info page, we need <a href="https://pythonhosted.org/Markdown/index.html">
-              Python-markdown</a>.
-              In the simplest case: <code>$ pip install markdown</code> OR  <code>$ easy_install markdown</code>
-            <li>Upload figure export script (need to be Admin):
-              <code style="width:700px">
-                $ cd figure/scripts   &nbsp &nbsp &nbsp # NB: important to be in this directory<br>
-                $ path/to/bin/omero script upload omero/figure_scripts/Figure_To_Pdf.py --official
+            <li>Optional: Figure legends can be formatted using Markdown syntax. To see this correctly in the exported PDF info page, we need <a href="https://pythonhosted.org/Markdown/index.html">Python-markdown</a> installed.<br>
+              <code style="width:650px">
+                $ pip install markdown
               </code>
             </li>
           </ul>
         </p>
-
         <h1>Update instructions</h1>
+        <h3>Upgrade the app</h3>
         <p>
+          Stop OMERO.web.
+          To upgrade a version of OMERO.figure prior to <strong>2.0.1</strong>:
           <ul>
             <li>
-              To update an existing install of OMERO.figure to the latest release, download it,
-              unzip and replace the contents of the current /figure
-              directory with the contents of the unzipped directory.
+              Remove the 'figure' directory from <strong>within</strong> the directory that is on your PYTHONPATH.
             </li>
-            <li>NB: If you are upgrading from OMERO.figure 1.0.0 to 1.1.0 or later,
-                note that we now use Python-Markdown
-                (optional, see install instructions above).</li>
-            <li>Replace PDF generation script (need to be Admin):
-              <code style="width:700px">
-                $ cd figure/scripts   &nbsp &nbsp &nbsp # NB: important to be in this directory<br>
-                # List scripts to get the ID of omero/figure_scripts/Figure_To_Pdf.py<br>
-                $ path/to/bin/omero script list<br>
-                $ path/to/bin/omero script replace &ltscriptId&gt omero/figure_scripts/Figure_To_Pdf.py
+            <li>Install the app using <a href="https://pip.pypa.io/en/stable/">pip</a>:<br>
+              <code style="width:650px">
+                $ pip install omero-figure
               </code>
             </li>
-            <li>If upgrading from OMERO.figure 1.0.0 to 1.1.0 or later, you can update the
+            <li>Remove the old 'figure' link and register the app:<br>
```

Comment:
The 'link' here is confusing since changing the 'link' is described below. Here we are updating because the app name changed. Maybe "Remove the old 'figure' app and add back the re-named 'omero-figure' app:"